packages/next/src/build/webpack/alias/react-dom-server-browser-experimental.js

```diff
@@ -0,0 +1,18 @@
+var l, s
+if (process.env.NODE_ENV === 'production') {
```

Comment:
did you copy this by hand? can't we automate something with taskr?

Reply:
I think this file can be used in both dev and prod so this is expected?